.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,51 @@
+---
+name: Bug Report
+about: Report a bug encountered while developing with Uno
+labels:
+- kind/bug
+- triage/untriaged
+---
+
+<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!
+
+If the matter is security related, please disclose it privately via https://github.com/nventive/Uno/security/
```

Comment:
Went to the link, looks like there's some setup required on our side

Reply:
Raised and tracking over at https://github.com/nventive/Uno/pull/976